doc/cascadia/profiles.schema.json

```diff
@@ -873,11 +873,29 @@
             "string",
             "null"
           ]
+        },
+        "commands":{
+          "description": "list of commands to execute",
+          "items": {
+            "$ref": "#/definitions/Keybinding/properties/command"
+          },
+          "minItems": 1,
+          "type": "array"
         }
       },
-      "required": [
-        "command",
-        "keys"
+      "allOf":[
+        {
+          "anyOf":[
+            {"required":["command"]},
+            {"required":["commands"]}
+          ]
+        },
+        {
+          "anyOf":[
+            {"required":["keys"]},
+            {"required":["name"]}
+          ]
+        }
```

Comment:
I think this is wrong. An entry _needs_ either...
- `commands` and `name`: all nested commands need a name, otherwise they can't be added.
- `command`: anything else is extra.
   - if it's just `command`, we auto-generate a name and add it to the command palette
   - if `name` is present, do the same as above, except with the given name
   - if `keys` is present, also bind the key chord to the action
   - if `icon` is present, add an icon to the command palette entry

Reply:
so just this?:
```json
"anyOf":[
        {"required":["commands","name"]},
        {"required":["command"]}
      ],
```

Reply:
yup!